src/components/prescriptions/PrescriptionButtonComponent.vue

```diff
@@ -0,0 +1,284 @@
+<template>
+    <div
+        class="prescriptionBox"
+        @submit.prevent="addTest">
+        <h3 class="prescriptionBox__header">Dodaj receptę:</h3>
+        <form class="prescriptionBox__form prescriptionForm">
+            <div class="selectBox">
+                <label for="addManually">Dodaj ręcznie</label>
+                <input
+                    id="addManually"
+                    v-model="addManually"
+                    type="checkbox" />
+            </div>
+            <div class="selectBox">
+                <label for="import">Zaimportuj receptę</label>
+                <input
+                    id="import"
+                    v-model="importFlag"
+                    type="checkbox" />
+            </div>
+
+            <div v-show="addManually">
```

Comment:
you should add logic when user click checkbox addmanually, box with import file should be hidden